controllers/deviceprofile_syncer.go

```diff
@@ -175,34 +169,34 @@ func getKubeNameWithPrefix(edgeName, NodePoolName string) string {
 }
 
 // createDeviceProfile creates the list of device profiles
-func createDeviceProfile(log logr.Logger, cli client.Client, edgeXDevs []devicev1alpha1.DeviceProfile, NodePoolName string) error {
+func createDeviceProfile(cli client.Client, edgeXDevs []devicev1alpha1.DeviceProfile, NodePoolName string) error {
```

Comment:
NodePoolName can be retrieved from opts

Reply:
NodePoolName has been retrieved from opts